File: Chapter-2/3/quote.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    string quote, speaker;
    cout << "What is the quote? ";
    getline(cin, quote);
    cout << "Who said it? ";
    getline(cin, speaker);
    cout << speaker << " says, " << "\"" << quote << "\"" << endl;
    return 0;
}

File: Chapter-2/3/quote.js
// node quote.js

const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
})

rl.question('What is the quote? ', (quote) => {
  rl.question('Who said it? ', (speaker) => {
    console.log('%s says, "%s"', speaker, quote);
    rl.close();
  })
})
